test/propagation.test.ts

```diff
@@ -102,15 +94,18 @@ describe('propagation', () => {
     const newContext = propagation.extract(context.active(), incomingCarrier);
     tracer.startSpan('request handler', {}, newContext).end();
 
-    spanProcessor.forceFlush().then(() => {
```

Comment:
let's make the whole test function an async one and flatten this callback, like so:
```js
await spanProcessor.forceFlush();

assert(exporter.getFinishedSpans().length == 1);
// ...

stopTracing();
```

Reply:
I'll happily do that. Just didn't want to change more than I absolutely have to.